src/com/lftechnology/task1/jan12/SentenceManipulation.java

```diff
@@ -0,0 +1,64 @@
+package com.lftechnology.task1.jan12;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+
+/**
+ * Q1.Takes the words in its argument list and prints out any duplicate words, the number of distinct words, and a list of the words with
+ * duplicates eliminated. Hint: Try with HashSet and List implementation.
+ * 
+ * @author Sanjib<sanjibmaharjan@lftechnology.com>
+ * 
+ */
+public class SentenceManipulation {
+
+	public static void main(String[] args) {
+		String word = "hello this is testing testing hello";
+		Map<String, Set<String>> result = distinguishDuplicates(word);
+
+		for (Entry<String, Set<String>> entry : result.entrySet()) {
+			// String key = entry.getKey();
+			// String value = entry.getValue();
+			System.out.println(entry.getKey() + " : " + Utils.listToString(entry.getValue()));
```

Comment:
Try not to put commented lines of code while sending pull requests. Also, Why is sysout here instead of Logger?